wiki/Beatmap_Editor/Kiai_Time/en.md

```diff
@@ -1,20 +1,24 @@
-# Kiai time
+---
+tags:
+  - timing section
+---
 
-![Stars shower the screen indicating Kiai Time has started.](img/Kiai_Time_test.jpg "Stars shower the screen indicating Kiai Time has started.")
+# Kiai Time
 
-![Kiai Time shows in the upper-left of the screen when active during edit mode.](img/Kiai_Time_edit.jpg "Kiai Time shows in the upper-left of the screen when active during edit mode.")
+*For regulations surrounding kiai time, see the [Ranking Criteria](/wiki/Ranking_Criteria).*
```

Comment:
> Is kiai time a proper noun or not? this article is inconsistent in its capitalisation

Reply:
I don't think it is. I've updated the article